examples/request_configuration.py

```diff
@@ -15,14 +15,26 @@
 
 """Example of request config"""
 from tensorbay import GAS
-from tensorbay.client.requests import config
-from tensorbay.opendataset import LISATrafficLight
+from tensorbay.client import config
+from tensorbay.dataset import Data, Dataset
 
+# Set "config" before authorize a client object as needede
```

Comment:
`before authorizing`
Should `as needed` be moved after `Set "config"`?
And a period is needed at the end of the line.